frontend/src/components/NavBar.jsx

```diff
@@ -25,7 +25,7 @@ const NavBar = () => {
         }`}
       >
         <div className="container-fluid">
-          <Link className="navbar-brand mb-0 h1" to="/draft2020-12">
+          <Link className="navbar-brand mb-0 h1" to="/">
```

Comment:
(I haven't tried this, I'm just reading it) but this is sending the homepage to `/` right, but where is it that makes that render  or redirect to `/dialects/draft2020-12`, and/or what's the homepage going to show after this?

Reply:
Routes definitions are located in `index.jsx` file, see call of `createHashRouter()` function. Basically, it should render the same thing it used to render before, which is the report page for `draft2020-12` dialect. There are no redirects, but ofc we could do this the other way around - navigating to `bowtie.report` could always redirect to `bowtie.report#/dialects/draft2020-12`. Either approaches are fine I guess

Reply:
Ah I see this change is irrelevant to all that, this is just tweaking some unrelated thing about whether we link to the homepage or to draft2020-12 from the navbar, I see (and yeah this way seems more correct to me).